### PR TITLE
Split array largest sum #714 Solved

### PR DESCRIPTION
**Pull request by krishna-NIT**
Split Array Largest Sum in JAVA
Done in JAVA #714 Done
@rudrakshi99 @nakul-19 @plazzy99 Done, Kindle Review it

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@yasharth291 Please Review my Code

**Comment by krishna-NIT** [comment]
@yasharth291 Please review it because Daily I get Conflicts in Readme File, I have to Resolve them Daily, Am I making any Mistake in Readme ?, can you please tell me why I am Getting Conflicts in Readme always, coz I am new to GIT and GITHUB

**Comment by krishna-NIT** [comment]
@Rohan27s @tarun26091999  Please Review it and Let me Know if any changes

File: Code/Java/splitarraylargestsum.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                7 2 5 10 8
+        Output: 18
+
+        Input : 3   // Length of Array
+                3   // value of m
+                1 4 4
+        Output: 9
+
+        Time Complexity: O(n)
+        Space Complexity: O(1)
```
Comment (Rohan27s) [inline Code/Java/splitarraylargestsum.java:74]: mention the value of n here

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Resolve the conflicts too

**Comment by krishna-NIT** [comment]
@Rohan27s  @rish-singhal Done with Changes!

**Pull request merged**